AppBuilder/platform/dataFields/ABFieldCombine.js

```diff
@@ -22,30 +24,29 @@ var ABFieldCombineComponent = new ABFieldComponent({
 
    elements: (App, field) => {
       ids = field.idsUnique(ids, App);
-      var L = App.Label;
 
       return [
          {
             id: ids.combinedFields,
             name: "combinedFields",
             view: "multicombo",
-            label: L("ab.combined.combinedFields", "*Combined Fields"),
-            labelWidth: App.config.labelWidthXLarge,
+            label: L("Combined Fields"),
+            labelWidth: this.AB.Config.labelWidthXLarge,
```

Comment:
verify these configs ... I think I might have told you wrong on how to reference them